### PR TITLE
Add receptorctl work list documentation

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

**Review by matoval — APPROVED** [review]

**Comment by thom-at-redhat** [comment]
@oraNod please review as time permits

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <<pid>``
+
+The output is divided into work unit sections listed below.
+Field values may be listed in their own section.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
Field values might be listed separately.
```

Comment (oraNod) [reply docs/source/receptorctl/receptorctl_work_list.rst]: May usually infers to "permission" while "might" infers possibility. Also "their own section" is slightly anthropomorphic and difficult for non-native speakers.

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      :widths: auto
+
+      * - Column
+        - Description
+      * - ``."Work unit string"``
+        - Random 8 character work unit (job) string.
+      * - ``."Work unit string"."Detail"``
+        - Work unit output.
+      * - ``."Work unit string"."ExtraData"``
+        - Additional information.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]: Is the additional info intended for debug purposes?

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+^^^^^^^^^^^^^^^^^
+
+.. list-table:: Work unit section
+      :header-rows: 1
+      :widths: auto
+
+      * - Column
+        - Description
+      * - ``."Work unit string"``
+        - Random 8 character work unit (job) string.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Random eight character work unit (job) string.
```

Comment (oraNod) [reply docs/source/receptorctl/receptorctl_work_list.rst]: The convention generally is to spell out numbers lower than 10.

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      * - ``."Work unit string"."Detail"``
+        - Work unit output.
+      * - ``."Work unit string"."ExtraData"``
+        - Additional information.
+      * - ``.""Work unit string"."State"``
+        - Current state for work unit (int).
+      * - ``.""Work unit string"."StateName"``
+        - Human readable current state for work unit.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Human-readable current state for the work unit.
```

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      * - Column
+        - Description
+      * - ``."Work unit string"``
+        - Random 8 character work unit (job) string.
+      * - ``."Work unit string"."Detail"``
+        - Work unit output.
+      * - ``."Work unit string"."ExtraData"``
+        - Additional information.
+      * - ``.""Work unit string"."State"``
+        - Current state for work unit (int).
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Current state for the work unit (int).
```

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      * - ``.""Work unit string"."StateName"``
+        - Human readable current state for work unit.
+      * - ``.""Work unit string"."StdoutSize"``
+        - Size of work unit output (bytes).
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Size of the work unit output (bytes).
```

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+      * - ``.""Work unit string"."WorkType"``
+        - Work unit type (queue).
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]: I'm not familiar enough with work units but what is `queue`? Is this a type? Are there others?

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+^^^^^^^^^^^^^^^^
+Work Unit States
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
Work unit states
```

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+      * - State
+        - StateName
+        - Description
+      * - ``0``
+        - ``Pending``
+        - Work unit has not started.
+      * - ``1``
+        - ``Running``
+        - Work unit is executing.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Work unit is currently executing.
```

File: docs/source/receptorctl/receptorctl_work_list.rst [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+      * - ``2``
+        - ``Succeeded``
+        - Work unit completed without error.
+      * - ``3``
+        - ``Failed``
+        - Work unit completed with error.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_list.rst]:
```suggestion
        - Work unit encountered an error or unexpected condition and did not complete.
```

File: docs/source/receptorctl/receptorctl_work_list.rst (hunk 2 above)
Comment (thom-at-redhat) [reply docs/source/receptorctl/receptorctl_work_list.rst]: Unclear atm

File: docs/source/receptorctl/receptorctl_work_list.rst (hunk 7 above)
Comment (thom-at-redhat) [reply docs/source/receptorctl/receptorctl_work_list.rst]: discussed directly and resolved

**Review by oraNod — APPROVED** [review]
LGTM. Thanks @thom-at-redhat

**Pull request merged**